managing-os/templates/fedora-minimal.md

```diff
@@ -52,7 +52,7 @@ The following table provides an overview of which packages are needed for which
 As expected, the required packages are to be installed in the running template with the following command. Replace "packages` with a space-delimited list of packages to be installed.
 
 ~~~
-[user@your-new-clone ~]$ sudo dnf install packages
+[user@your-new-clone ~]$ sudo dnf --enablerepo=qubes-vm-*-current-testing install packages
```

Comment:
This suggests that using the current-testing repo is required, but some users on stable systems may not want to install packages that are still in testing.

Reply:
current-testing is required for qubes-input-proxy-sender. It is worth to point this out, I wasted two hours trying to figure out what was wrong with my sys-usb because of this! :D

Reply:
It might be the case shortly after releasing first rc, but now all packages are available in stable repository.

Reply:
> current-testing is required for qubes-input-proxy-sender. It is worth to point this out, I wasted two hours trying to figure out what was wrong with my sys-usb because of this! :D

Yes, but my point still stands. It wouldn't be right to tell *every* user who wants to customize `fedora-minimal` to download testing packages.